plugin.video.fsgo/default.py

```diff
@@ -50,7 +52,7 @@ def addon_log(string):
         xbmc.log(msg=msg, level=xbmc.LOGDEBUG)
```

Comment:
@phil65 fsgo.py is Kodi independent and requires the 'debug' parameter so I can't really rely on xbmc.LOGDEBUG. I could lie and send True, but then the debug code (which can be a bit heavy in some cases) would consistently even if it's not getting printed in the log file. 

This feels like the best solution IMO.